File: chapter-01/minyoung.kim/rank.js
const Num = 10;

function main () {    
    const a = [56, 25, 57, 88, 100, 61, 55, 67, 76, 56];
    const rank = new Array(Num);
    let i;
    let j;

    for (i=0; i<Num; i++) {
        rank[i] = 1;
        for(j=0; j<Num; j++) {
            if(a[j] > a[i]) {
                rank[i]++;
            }
        }
    }    
    console.log('점수   순위');
    for(i=0; i<Num; i++) {
        console.log(a[i] + '    ' + rank[i]);
    }
    return '';
}

console.log(main());

File: chapter-01/jinsu.park/rank.py
num =10

a = [56, 25, 67,88,100,61,55,67,76,56]
rank = [0 for _ in range(10)]

for i in range(num): 
  rank[i]=1;
  
  for j in range(num):
    if a[j]>a[i]:
      rank[i]+=1 #파이썬은 후위 연산자가 뿔뿔이 안되네

print("점수 순위\n")
for i in range(num):
  print(a[i],rank[i])

print("파이썬은 들여쓰기가 참 중요합니다.")